src/Tribe/Aggregator/Record/Queue_Cleaner.php

```diff
@@ -6,10 +6,10 @@ class Tribe__Events__Aggregator__Record__Queue_Cleaner {
 	/**
 	 * Default is 12hrs.
 	 *
-	 * @var int The time a record is allowed to stall before havint its status set to to failed since its creation in
+	 * @var int The time a record is allowed to stall before have the status set to to failed since its creation in
 	 *          seconds.
 	 */
-	protected $time_to_live = 43200; // For pre-PHP 5.6 compat, we do not define as 12 * HOUR_IN_SECONDS
+	protected $time_to_live = HOUR_IN_SECONDS * 12; // For pre-PHP 5.6 compat, we do not define as 12 * HOUR_IN_SECONDS
```

Comment:
No need to keep pre-5.6 compat anymore.
```suggestion
	protected $time_to_live = HOUR_IN_SECONDS * 12;
```